spatialmath/geometry.go

```diff
@@ -107,7 +107,7 @@ func (config *GeometryConfig) ParseConfig() (Geometry, error) {
 	if err != nil {
 		return nil, err
 	}
-	offset := Compose(NewPoseFromOrientation(orientation), NewPoseFromPoint(config.TranslationOffset))
+	offset := Compose(NewPoseFromPoint(config.TranslationOffset), NewPoseFromOrientation(orientation))
```

Comment:
I think this can just be `offset := NewPose(config.TranslationOffset, orientation)`